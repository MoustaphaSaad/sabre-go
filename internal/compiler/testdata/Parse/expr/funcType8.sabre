func(a, _ int, z float32) bool