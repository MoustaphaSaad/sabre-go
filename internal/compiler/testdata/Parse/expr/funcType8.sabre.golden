(FuncType
  (FuncType-Parameters
    (IdentifierExpr IDENTIFIER(a))
    (IdentifierExpr IDENTIFIER(_))
    (NamedType IDENTIFIER(int))
    (IdentifierExpr IDENTIFIER(z))
    (NamedType IDENTIFIER(float32))
  )
  (FuncType-Results
    (NamedType IDENTIFIER(bool))
  )
)